apps/site/assets/ts/schedule/components/schedule-finder/UpcomingDepartures.tsx

```diff
@@ -139,7 +139,30 @@ export const UpcomingDepartures = ({
   if (journeys !== null && hasPredictions(journeys)) {
     return (
       <>
-        <h3>Upcoming Departures</h3>
+        <div className="schedule-table__upcoming-departures-header">
+          <h3>Upcoming Departures</h3>
+          &nbsp;
+          <span className="schedule-table__live-clock" aria-hidden>
+            <span className="icon-realtime animate">
+              <span className="icon-realtime animate notranslate c-svg__icon-live-clock">
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="9"
+                  height="9"
+                  role="img"
+                  viewBox="0 0 9 9"
+                >
+                  <title>clock</title>
+                  <path
+                    fillRule="evenodd"
+                    d="M5 4V1.5a.5.5 0 0 0-1 0v3a.498.498 0 0 0 .5.5h2a.5.5 0 0 0 0-1H5zm-.5 5a4.5 4.5 0 1 1 0-9 4.5 4.5 0 0 1 0 9z"
+                  />
+                </svg>
```

Comment:
Might it make sense to import/export the clock icon from our static assets, similarly to what's done [in this helper file](https://github.com/mbta/dotcom/blob/95846cf829daba6f4a97e54a153b7a622c6ab851/apps/site/assets/ts/helpers/icon.tsx), rather than duplicating the SVG code here?

Reply:
yes...I tried that and it turned into a whole thing so I moved on, but now that I have a working branch to compare it with it would make sense to try to refactor that again.